### ISSUE TITLE
Unable to resolve WSDL method name GetResourceUsage for namespace name urn:vim25

### ISSUE THREAD

**Issue by babyachievement** [yavijava/yavijava#181]
Hi, when I invoked the getResourceUsage method of ClusterComputeResource, I got the the errror like this:

``` java
Ignore ssl: true
Invoking method: RetrieveServiceContent
Marshalled Payload String xml: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><RetrieveServiceContent xmlns="urn:vim25"><_this type="ServiceInstance">ServiceInstance</_this></RetrieveServiceContent></soapenv:Body></soapenv:Envelope>
POST: null
Payload: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><RetrieveServiceContent xmlns="urn:vim25"><_this type="ServiceInstance">ServiceInstance</_this></RetrieveServiceContent></soapenv:Body></soapenv:Envelope>
Successfully fetched InputStream.
Cookie was null. Fetching Set-Cookie header to get new Cookie.
Converting xml response from server to: ServiceContent
Parsing XML payload from server. ServiceContent
XML Document: <?xml version="1.0" encoding="UTF-8"?>
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<soapenv:Body>
<RetrieveServiceContentResponse xmlns="urn:vim25"><returnval><rootFolder type="Folder">group-d1</rootFolder><propertyCollector type="PropertyCollector">propertyCollector</propertyCollector><viewManager type="ViewManager">ViewManager</viewManager><about><name>VMware vCenter Server</name><fullName>VMware vCenter Server 5.5.0 build-1312298</fullName><vendor>VMware, Inc.</vendor><version>5.5.0</version><build>1312298</build><localeVersion>INTL</localeVersion><localeBuild>000</localeBuild><osType>win32-x64</osType><productLineId>vpx</productLineId><apiType>VirtualCenter</apiType><apiVersion>5.5</apiVersion></about><setting type="OptionManager">VpxSettings</setting><userDirectory type="UserDirectory">UserDirectory</userDirectory><sessionManager type="SessionManager">SessionManager</sessionManager><authorizationManager type="AuthorizationManager">AuthorizationManager</authorizationManager><perfManager type="PerformanceManager">PerfMgr</perfManager><scheduledTaskManager type="ScheduledTaskManager">ScheduledTaskManager</scheduledTaskManager><alarmManager type="AlarmManager">AlarmManager</alarmManager><eventManager type="EventManager">EventManager</eventManager><taskManager type="TaskManager">TaskManager</taskManager><extensionManager type="ExtensionManager">ExtensionManager</extensionManager><customizationSpecManager type="CustomizationSpecManager">CustomizationSpecManager</customizationSpecManager><customFieldsManager type="CustomFieldsManager">CustomFieldsManager</customFieldsManager><diagnosticManager type="DiagnosticManager">DiagMgr</diagnosticManager><licenseManager type="LicenseManager">LicenseManager</licenseManager><searchIndex type="SearchIndex">SearchIndex</searchIndex><fileManager type="FileManager">FileManager</fileManager><virtualDiskManager type="VirtualDiskManager">virtualDiskManager</virtualDiskManager></returnval></RetrieveServiceContentResponse>
</soapenv:Body>
</soapenv:Envelope>
API Version detected: 5.5
Set soapAction to: urn:vim25/5.5
Invoking method: RetrieveServiceContent
Marshalled Payload String xml: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><RetrieveServiceContent xmlns="urn:vim25"><_this type="ServiceInstance">ServiceInstance</_this></RetrieveServiceContent></soapenv:Body></soapenv:Envelope>
POST: urn:vim25/5.5
Payload: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><RetrieveServiceContent xmlns="urn:vim25"><_this type="ServiceInstance">ServiceInstance</_this></RetrieveServiceContent></soapenv:Body></soapenv:Envelope>
Setting Cookie.
Successfully fetched InputStream.
Converting xml response from server to: ServiceContent
Parsing XML payload from server. ServiceContent
XML Document: <?xml version="1.0" encoding="UTF-8"?>
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<soapenv:Body>
<RetrieveServiceContentResponse xmlns="urn:vim25"><returnval><rootFolder type="Folder">group-d1</rootFolder><propertyCollector type="PropertyCollector">propertyCollector</propertyCollector><viewManager type="ViewManager">ViewManager</viewManager><about><name>VMware vCenter Server</name><fullName>VMware vCenter Server 5.5.0 build-1312298</fullName><vendor>VMware, Inc.</vendor><version>5.5.0</version><build>1312298</build><localeVersion>INTL</localeVersion><localeBuild>000</localeBuild><osType>win32-x64</osType><productLineId>vpx</productLineId><apiType>VirtualCenter</apiType><apiVersion>5.5</apiVersion><instanceUuid>73868C24-9E86-4EB3-9C35-988C75177A42</instanceUuid><licenseProductName>VMware VirtualCenter Server</licenseProductName><licenseProductVersion>5.0</licenseProductVersion></about><setting type="OptionManager">VpxSettings</setting><userDirectory type="UserDirectory">UserDirectory</userDirectory><sessionManager type="SessionManager">SessionManager</sessionManager><authorizationManager type="AuthorizationManager">AuthorizationManager</authorizationManager><serviceManager type="ServiceManager">ServiceMgr</serviceManager><perfManager type="PerformanceManager">PerfMgr</perfManager><scheduledTaskManager type="ScheduledTaskManager">ScheduledTaskManager</scheduledTaskManager><alarmManager type="AlarmManager">AlarmManager</alarmManager><eventManager type="EventManager">EventManager</eventManager><taskManager type="TaskManager">TaskManager</taskManager><extensionManager type="ExtensionManager">ExtensionManager</extensionManager><customizationSpecManager type="CustomizationSpecManager">CustomizationSpecManager</customizationSpecManager><customFieldsManager type="CustomFieldsManager">CustomFieldsManager</customFieldsManager><diagnosticManager type="DiagnosticManager">DiagMgr</diagnosticManager><licenseManager type="LicenseManager">LicenseManager</licenseManager><searchIndex type="SearchIndex">SearchIndex</searchIndex><fileManager type="FileManager">FileManager</fileManager><datastoreNamespaceManager type="DatastoreNamespaceManager">DatastoreNamespaceManager</datastoreNamespaceManager><virtualDiskManager type="VirtualDiskManager">virtualDiskManager</virtualDiskManager><snmpSystem type="HostSnmpSystem">SnmpSystem</snmpSystem><vmProvisioningChecker type="VirtualMachineProvisioningChecker">ProvChecker</vmProvisioningChecker><vmCompatibilityChecker type="VirtualMachineCompatibilityChecker">CompatChecker</vmCompatibilityChecker><ovfManager type="OvfManager">OvfManager</ovfManager><ipPoolManager type="IpPoolManager">IpPoolManager</ipPoolManager><dvSwitchManager type="DistributedVirtualSwitchManager">DVSManager</dvSwitchManager><hostProfileManager type="HostProfileManager">HostProfileManager</hostProfileManager><clusterProfileManager type="ClusterProfileManager">ClusterProfileManager</clusterProfileManager><complianceManager type="ProfileComplianceManager">MoComplianceManager</complianceManager><localizationManager type="LocalizationManager">LocalizationManager</localizationManager><storageResourceManager type="StorageResourceManager">StorageResourceManager</storageResourceManager><guestOperationsManager type="GuestOperationsManager">guestOperationsManager</guestOperationsManager></returnval></RetrieveServiceContentResponse>
</soapenv:Body>
</soapenv:Envelope>
Invoking method: Login
Marshalled Payload String xml: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><Login xmlns="urn:vim25"><_this type="SessionManager">SessionManager</_this><userName>oneapm</userName><password>1qaz@WSX</password></Login></soapenv:Body></soapenv:Envelope>
POST: urn:vim25/5.5
Payload: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><Login xmlns="urn:vim25"><_this type="SessionManager">SessionManager</_this><userName>oneapm</userName><password>1qaz@WSX</password></Login></soapenv:Body></soapenv:Envelope>
Setting Cookie.
Successfully fetched InputStream.
Converting xml response from server to: UserSession
Parsing XML payload from server. UserSession
XML Document: <?xml version="1.0" encoding="UTF-8"?>
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<soapenv:Body>
<LoginResponse xmlns="urn:vim25"><returnval><key>52b42c0a-cc59-da2c-9137-bee4062853a0</key><userName>oneapm</userName><fullName>oneapm </fullName><loginTime>2016-08-19T06:48:52.230036Z</loginTime><lastActiveTime>2016-08-19T06:48:52.230036Z</lastActiveTime><locale>zh_CN</locale><messageLocale>zh_CN</messageLocale><extensionSession>false</extensionSession><ipAddress>10.128.7.212</ipAddress><userAgent>Java/1.7.0_03</userAgent><callCount>0</callCount></returnval></LoginResponse>
</soapenv:Body>
</soapenv:Envelope>
VCenterPluginSession:class com.oneapm.si.rm.plugin.vimapi.vcenter.VCenterClusterComputeResourceStatement begin to execute
API version >= 4 detected. Using buildFullTraversalV4.
Invoking method: RetrieveProperties
Marshalled Payload String xml: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><RetrieveProperties xmlns="urn:vim25"><_this type="PropertyCollector">propertyCollector</_this><specSet><propSet><type>ClusterComputeResource</type><all>false</all><pathSet>name</pathSet></propSet><objectSet><obj type="Folder">group-d1</obj><skip>false</skip><selectSet xsi:type="TraversalSpec"><name>visitFolders</name><type>Folder</type><path>childEntity</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet><selectSet><name>dcToHf</name></selectSet><selectSet><name>dcToVmf</name></selectSet><selectSet><name>dcToDs</name></selectSet><selectSet><name>dcToNetf</name></selectSet><selectSet><name>crToH</name></selectSet><selectSet><name>crToRp</name></selectSet><selectSet><name>HToVm</name></selectSet><selectSet><name>rpToVm</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToDs</name><type>Datacenter</type><path>datastoreFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToNetf</name><type>Datacenter</type><path>networkFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>vAppToRp</name><type>VirtualApp</type><path>resourcePool</path><skip>false</skip><selectSet><name>rpToRp</name></selectSet><selectSet><name>vAppToRp</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToVmf</name><type>Datacenter</type><path>vmFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToHf</name><type>Datacenter</type><path>hostFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>crToH</name><type>ComputeResource</type><path>host</path><skip>false</skip></selectSet><selectSet xsi:type="TraversalSpec"><name>crToRp</name><type>ComputeResource</type><path>resourcePool</path><skip>false</skip><selectSet><name>rpToRp</name></selectSet><selectSet><name>rpToVm</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>rpToRp</name><type>ResourcePool</type><path>resourcePool</path><skip>false</skip><selectSet><name>rpToRp</name></selectSet><selectSet><name>rpToVm</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>HToVm</name><type>HostSystem</type><path>vm</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>rpToVm</name><type>ResourcePool</type><path>vm</path><skip>false</skip></selectSet></objectSet></specSet></RetrieveProperties></soapenv:Body></soapenv:Envelope>
POST: urn:vim25/5.5
Payload: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><RetrieveProperties xmlns="urn:vim25"><_this type="PropertyCollector">propertyCollector</_this><specSet><propSet><type>ClusterComputeResource</type><all>false</all><pathSet>name</pathSet></propSet><objectSet><obj type="Folder">group-d1</obj><skip>false</skip><selectSet xsi:type="TraversalSpec"><name>visitFolders</name><type>Folder</type><path>childEntity</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet><selectSet><name>dcToHf</name></selectSet><selectSet><name>dcToVmf</name></selectSet><selectSet><name>dcToDs</name></selectSet><selectSet><name>dcToNetf</name></selectSet><selectSet><name>crToH</name></selectSet><selectSet><name>crToRp</name></selectSet><selectSet><name>HToVm</name></selectSet><selectSet><name>rpToVm</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToDs</name><type>Datacenter</type><path>datastoreFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToNetf</name><type>Datacenter</type><path>networkFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>vAppToRp</name><type>VirtualApp</type><path>resourcePool</path><skip>false</skip><selectSet><name>rpToRp</name></selectSet><selectSet><name>vAppToRp</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToVmf</name><type>Datacenter</type><path>vmFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>dcToHf</name><type>Datacenter</type><path>hostFolder</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>crToH</name><type>ComputeResource</type><path>host</path><skip>false</skip></selectSet><selectSet xsi:type="TraversalSpec"><name>crToRp</name><type>ComputeResource</type><path>resourcePool</path><skip>false</skip><selectSet><name>rpToRp</name></selectSet><selectSet><name>rpToVm</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>rpToRp</name><type>ResourcePool</type><path>resourcePool</path><skip>false</skip><selectSet><name>rpToRp</name></selectSet><selectSet><name>rpToVm</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>HToVm</name><type>HostSystem</type><path>vm</path><skip>false</skip><selectSet><name>visitFolders</name></selectSet></selectSet><selectSet xsi:type="TraversalSpec"><name>rpToVm</name><type>ResourcePool</type><path>vm</path><skip>false</skip></selectSet></objectSet></specSet></RetrieveProperties></soapenv:Body></soapenv:Envelope>
Setting Cookie.
Successfully fetched InputStream.
Converting xml response from server to: ObjectContent[]
Parsing XML payload from server. ObjectContent[]
XML Document: <?xml version="1.0" encoding="UTF-8"?>
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<soapenv:Body>
<RetrievePropertiesResponse xmlns="urn:vim25"><returnval><obj type="ClusterComputeResource">domain-c227</obj><propSet><name>name</name><val xsi:type="xsd:string">testcluster</val></propSet></returnval><returnval><obj type="ClusterComputeResource">domain-c278</obj><propSet><name>name</name><val xsi:type="xsd:string">dfsfs</val></propSet></returnval></RetrievePropertiesResponse>
</soapenv:Body>
</soapenv:Envelope>
Invoking method: GetResourceUsage
Marshalled Payload String xml: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><GetResourceUsage xmlns="urn:vim25"><_this type="ClusterComputeResource">domain-c227</_this></GetResourceUsage></soapenv:Body></soapenv:Envelope>
POST: urn:vim25/5.5
Payload: <?xml version="1.0" encoding="UTF-8"?><soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><soapenv:Body><GetResourceUsage xmlns="urn:vim25"><_this type="ClusterComputeResource">domain-c227</_this></GetResourceUsage></soapenv:Body></soapenv:Envelope>
Setting Cookie.
Caught an IOException. Reading ErrorStream for results.
java.io.IOException: Server returned HTTP response code: 500 for URL: https://10.128.9.63/sdk
    at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1612)
    at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)
    at com.vmware.vim25.ws.WSClient.getInputStreamFromConnection(WSClient.java:187)
    at com.vmware.vim25.ws.WSClient.post(WSClient.java:174)
    at com.vmware.vim25.ws.WSClient.invoke(WSClient.java:94)
    at com.vmware.vim25.ws.VimStub.getResourceUsage(VimStub.java:5213)
    at com.vmware.vim25.mo.ClusterComputeResource.getResourceUsage(ClusterComputeResource.java:183)
    at com.oneapm.si.rm.plugin.vimapi.vcenter.VCenterClusterComputeResourceStatement.buildResultSet(VCenterClusterComputeResourceStatement.java:56)
    at com.oneapm.si.rm.plugin.vimapi.vcenter.VCenterClusterComputeResourceStatement.execute(VCenterClusterComputeResourceStatement.java:35)
    at com.oneapm.si.rm.plugin.vimapi.vcenter.VCenterPluginSession.execute(VCenterPluginSession.java:124)
    at com.oneapm.si.rm.plugin.vimapi.vcenter.VCenterPluginSessionTest.testCluster(VCenterPluginSessionTest.java:151)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:601)
    at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
    at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
    at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
    at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
    at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
    at org.junit.runners.BlockJUnit4ClassRunner.runNotIgnored(BlockJUnit4ClassRunner.java:79)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:71)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
    at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
    at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
    at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
    at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
    at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
    at org.junit.runner.JUnitCore.run(JUnitCore.java:157)
    at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:119)
    at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42)
    at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:234)
    at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:74)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:601)
    at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Converting xml response from server to: ClusterResourceUsageSummary
Parsing XML payload from server. ClusterResourceUsageSummary
XML Document: <?xml version="1.0" encoding="UTF-8"?>
<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<soapenv:Body>
<soapenv:Fault><faultcode>ServerFaultCode</faultcode><faultstring>Unable to resolve WSDL method name GetResourceUsage for namespace name urn:vim25

while parsing SOAP body
at line 1, column 271

while parsing SOAP envelope
at line 1, column 38

while parsing HTTP request before method was determined
at line 1, column 0</faultstring><detail><InvalidRequestFault xmlns="urn:vim25" xsi:type="InvalidRequest"></InvalidRequestFault></detail></soapenv:Fault>
</soapenv:Body>
</soapenv:Envelope>
Exception caught while invoking method: GetResourceUsage
com.vmware.vim25.InvalidRequest: Unable to resolve WSDL method name GetResourceUsage for namespace name urn:vim25

while parsing SOAP body
at line 1, column 271

while parsing SOAP envelope
at line 1, column 38

while parsing HTTP request before method was determined
```

**Comment by vThinkBeyondVM** [comment]
@babyachievement some time back I had written one sample on this API : http://vthinkbeyondvm.com/how-to-quickly-get-vsphere-cluster-resource-usage-cpu-mem-storage-using-api-cool-api-in-vsphere-6-0/

It worked fine for me. I am using 6.0.03 (yavijava jars).

**Comment by babyachievement** [comment]
@vThinkBeyondVM ,I have already tried the sample, but it did not work for me, I still got the error:

``` java
Exception in thread "main" com.vmware.vim25.InvalidRequest: Unable to resolve WSDL method name GetResourceUsage for namespace name urn:vim25
```

should I modify the config file of VCenter? 

The version of our vcenter is 5.5.0.

**Comment by babyachievement** [comment]
@vThinkBeyondVM, I'm  very sorry. I did not notice that, GetResourceUsage just support Vsphere 6.0.

**Comment by vThinkBeyondVM** [comment]
Yeah. It is only available as part of vCenter 6.0

**Comment by babyachievement** [comment]
@vThinkBeyondVM, thank you.
